src/main/java/com/gnerga/app/remote/model/GitHubResponseDto.java

```diff
@@ -0,0 +1,17 @@
+package com.gnerga.app.remote.model;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@Builder
+@Data
+public class GitHubResponseDto {
+
+    final String full_name;
```

Comment:
zamiast pisac _ to mozemy bardziej elegancko @JsonProperty("full_name") i wtedy zmienna normalnie CamelCase